docs/library/sequencing/clockdivider/example-0.py

```diff
@@ -0,0 +1,23 @@
+from signalflow import *
+graph = AudioGraph()
+
+#-------------------------------------------------------------------------------
+# Using a ClockDivider to create rhythms related to the main clock. Here the oscillator in the left channel is heard on every tick of the clock. The oscillator in the right channel is heard every 3 ticks of the clock.
+#-------------------------------------------------------------------------------
+clock = Impulse(2.0)
+divided_clock = ClockDivider(clock, 3)
+
+oscillator_a = TriangleOscillator(220)
+oscillator_b = TriangleOscillator(440)
+
+envelope_a = ASREnvelope(0.01, 0.0, 0.25, 1.0, clock)
+envelope_b = ASREnvelope(0.01, 0.0, 0.5, 1.0, divided_clock)
+
+voice_a = oscillator_a * envelope_a
+voice_b = oscillator_b * envelope_b
+
+left = StereoPanner(voice_a, -1.0)
```

Comment:
I love this example!
To be slightly pedantic again - this formulation of "voice_a through left channel and voice_b through right channel" could (slightly more efficiently) be implemented using `ChannelArray(voice_a, voice_b)`. But I like the use of `StereoPanner` here, so maybe you could do `StereoPanner(voice_a, -0.5)` and `StereoPanner(voice_b, 0.5)` to do something that couldn't be achieved with a ChannelArray.

Reply:
Also missing a graph.wait()?

Reply:
Great catch re: graph.wait()!
Good idea re: StereoPanner -- I'll change the pan values to -0.75 and 0.75 to give a bit more stereo separation, which should allow for easy distinction between the two voices whilst still doing something ChannelArray can't. Feel like with -0.5/0.5 the distinction is less clear and doesn't quite demonstrate the ClockDivider as effectively, if you get me?